obstacle_check_angle: 0.04
obstacle_range_mix: 2.5
scan_range_threshold: 10
branch_diff_x_min: 1
branch_diff_x_max: 10
branch_diff_y_min: 0.25
branch_diff_y_max: 10
